InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -199,6 +199,11 @@ public void periodic(float deltaTime) {
         }
     }
 
+    public void spinUp(double velocity_rpm) {
+        SmartDashboard.putNumber(getName() + "/Shooter Velocity Rpm", velocity_rpm);

```

Comment:
We don’t want to use the network tables to communicate between functions. We should really minimize network table use.